db/migrate/20150709221755_create_logins.rb

```diff
@@ -1,4 +1,4 @@
-class CreateLogins < ActiveRecord::Migration
+class CreateLogins < ActiveRecord::Migration[4.2]
```

Comment:
I discovered that from the failing Travis build. The link I pasted a minute ago suggests conditional statement in the migration like so: https://github.com/thoughtbot/clearance/commit/b0ce189c69d52155660232604cc30d729d6b4221, that's not a great solution though. Rails should support the syntax in 4.2+ since that's what the error message shows as usage.